src/SUMMARY.md

```diff
@@ -31,6 +31,7 @@
 
 - [Introduction](./contributing.md)
 - [About the compiler team](./compiler-team.md)
+- [Using git](./git.md)
```

Comment:
Capitalize?

```suggestion
- [Using Git](./git.md)
```

Reply:
(It will require some changes to `src/git.md`)

Reply:
At some point I got into the habit of using "git" with lowercase because I saw it done elsewhere and I guess I assumed that this is what they wanted. It's definitely wrong though, and I'll change uses to uppercase across the board